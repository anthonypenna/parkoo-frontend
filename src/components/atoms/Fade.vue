<template>
  <transition name="fade">
    <slot />
  </transition>
</template>

<style lang="scss" scoped>
.fade-enter-active,
.fade-leave-active {
  transition: opacity 0.4s ease-out;
}

.fade-enter {
  opacity: 0;
}

.fade-enter-to {
  opacity: 1;
}

.fade-leave {
  opacity: 1;
}

.fade-leave-to {
  opacity: 0;
}
</style>
