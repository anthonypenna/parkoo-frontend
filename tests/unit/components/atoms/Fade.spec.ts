import { shallowMount } from '@vue/test-utils'
import Fade from '@/components/atoms/Fade.vue'

describe('<Fade />', () => {
  it('should render correctly', () => {
    const wrapper = shallowMount(Fade)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
