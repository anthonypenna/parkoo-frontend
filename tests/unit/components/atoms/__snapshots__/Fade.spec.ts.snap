// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Fade /> should render correctly 1`] = `<transition-stub name="fade"></transition-stub>`;
